test-vectors/presentation_exchange/validate_submission.json

```diff
@@ -0,0 +1,243 @@
+{
+  "description":"Validate submission",
+  "vectors":[
```

Comment:
These vectors also don't have output. What would an `"errors": false` represent for these vectors?

Reply:
output is optional, errors false means that this input does not throw an exception